internal/operators/lvm/config.go

```diff
@@ -16,7 +16,7 @@ const (
 
 type Config struct {
 	LvmCPUPerHost          int64  `envconfig:"LVM_CPU_PER_HOST" default:"1"`
-	LvmMemoryPerHostMiB    int64  `envconfig:"LVM_MEMORY_PER_HOST_MIB" default:"1200"`
+	LvmMemoryPerHostMiB    int64  `envconfig:"LVM_MEMORY_PER_HOST_MIB" default:"400"`
```

Comment:
You might need to dynamically set `LvmMemoryPerHostMiB` on 4.13 and not for other OCP versions.
See [here](https://github.com/openshift/assisted-service/blob/a3651ed5446a8dd40161195372327c487a8f73cd/internal/operators/lvm/lvm_operator.go#L154)

Reply:
You can do something like this 
```go
func (o *operator) getLvmMemoryPerHostMib(cluster *common.Cluster) int64 {
	if greaterOrEqual, _ := common.BaseVersionGreaterOrEqual("4.13", cluster.OpenshiftVersion); !greaterOrEqual {
		return o.config.LvmMemoryPerHostMiB
	} 
	
	return o.config.UpdatedLvmMemoryPerHostMiB
}
```

Just give UpdatedLvmMemoryPerHostMiB a better name :)

Reply:
Note: I think that the second option is better (update and not add)